store.go

```diff
@@ -1,28 +1,49 @@
 package main
 
+import (
+	"context"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
 type ToDoStore struct {
-	Tasks map[int]*Task
-	Count int // total number of tasks (used as ID for individual task)
+	Tasks       map[int]*Task
+	IDCounter   int // (ID given to a task before incrementing by 1)
```

Comment:
I like the change to rename this from `Count` to include `ID` in the name. This more closely matches how its used and helps me (as an outsider to your code) understand how this value is used.